cdk/lib/salesforce-disaster-recovery.ts

```
@@ -191,6 +193,7 @@ export class SalesforceDisasterRecovery extends GuStack {
 				stateJson: {
 					Type: 'Map',
 					MaxConcurrency: 10,
+					ToleratedFailurePercentage: 100,
```

Maybe put a comment here to explain why we are ignoring errors?

---

Good point - I have added a comment